src/Components/test/E2ETest/ServerExecutionTests/JsInitializersTest.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using BasicTestApp;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.E2ETesting;
+using OpenQA.Selenium;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests
+{
+    public class ServerJsInitializersTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>>
```

Comment:
This looks the same as `JsInitializersTest`. Does it work to subclass it for the server, like we do for most of the E2E tests?

Reply:
Ah, copy paste stuff, I'll fix it.